src/Psr7/RequestTransformer.php

```diff
@@ -15,12 +15,6 @@ class RequestTransformer
      */
     public function toPsr7(Request $request)
     {
-        // Prevent DiactorosFactory::createRequest from locking
-        // the content out from subsequent Request client code. This hack
-        // is needed until https://github.com/symfony/symfony/pull/19549
-        // finds its way into a future Symfony release.
-        $request->getContent();
```

Comment:
This comment is somewhat misleading, that line should be kept and the comment changed or deleted altogether.

The reason is that while this PR was merged into Symfony 2.7.17 and is no longer required if you use an up-to-date version of Symfony, the bundle must still work with the first 16 minor releases of Symfony 2.7.